src/mocks/MockFinanceData.ts

```
@@ -37,33 +37,42 @@ export class MockFinanceData {
+      },
+      { symbol: 'ADBE', displaySymbol: 'Adobe Inc.', description: 'Adobe Inc.' },
+      { symbol: 'INTC', displaySymbol: 'Intel Corporation', description: 'Intel Corporation' }
     ];
   }
 
-  static createDataBySymbol(symbol: string): HistoricalData {
+  static createDataBySymbol(_symbol: string): HistoricalData {
+    console.log('createDataBySymbol', _symbol);
     return {
       date: new Date().getTime(),
       open: this.getRandomIntInRange(this.PRICE_RANGES.OPEN.min, this.PRICE_RANGES.OPEN.max),
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)